### ISSUE TITLE
Pre-training

### ISSUE THREAD

**Issue by BakingBrains** [b04901014/FT-w2v2-ser#5]
```
python run_pretrain.py --datadir Audio_Dir \
                       --labelpath Label_Path
                       --labeling_method hard \
                       --saving_path Saving_Path \
                       --training_step 10000 \
                       --save_top_k 1 \
                       --wav2vecpath Wav2vecCKPT \
                       --precision 16
```

Here --labelpath, should be a session label or metalabel.json?

**Comment by b04901014** [comment]
In my experiment I only use session label since I don't want the test set to be revealed during pre-training, `metalabel.json` is simply a by-product.
But if you don't care, you can feed `metalabel.json` into it, it will pre-train on all of the IEMOCAP data regardless of the splits and the categories.

**Comment by BakingBrains** [comment]
@b04901014 Thank you
